tx-pool/src/component/pending.rs

```diff
@@ -174,7 +178,15 @@ mod tests {
             tx3.proposal_short_id(),
             tx1.proposal_short_id(),
         ];
-        assert_eq!(txs_sorted_by_fee_rate, expect_result);
+        assert_eq!(txs_sorted_by_fee_rate, expect_result.clone());
+
+        let sorted_fee_and_relation = pool
+            .sorted_fee_and_relation()
+            .iter()
+            .map(|key| key.id.clone())
+            .collect::<Vec<_>>();
+
+        assert_eq!(sorted_fee_and_relation, expect_result);
```

Comment:
we can't guarantee this after sorting, this assertion may be broken with a different data set.

Reply:
Please copy the data and write another test case. These two functions are unrelated.

Reply:
updated.